web-animations-2/Overview.bs

```diff
@@ -2366,15 +2367,29 @@ Animation Triggers {#triggers}
 ### Overview ### {#triggers-overview}
 
 An <dfn export>animation trigger</dfn> is used to control the playback
-of its associated [=animation=] for time-driven animations.
-Like animations, [=animation triggers=] are associated with a [=timeline=]
+of its associated [=animation=] for time-driven animations. There are two
+types of [=animation triggers=]:
+<dfn export lt='timeline-based animation trigger'>timeline-based</dfn> and
+<dfn export lt='event-based animation trigger'>event-based</dfn>.
+
+Like animations, [=timeline-based animation triggers=] are associated with a [=timeline=]
 and are attached to an [=animation attachment range|attachment range=].
+Changes to animation playback occur when the associated [=timeline=] transitions
+between inside and outside the [=animation attachment range|attachment range=].
+
+An [=event-based animation trigger=] is associated with
+an <dfn export for="animation trigger">event type</dfn>
+and an <dfn export >animation trigger name</dfn>.
+Changes to animation playback occur when an {{Event!!interface}} is dispatched
+with {{Event/type}} matching the trigger's associated [=animation trigger/event type=]
+and {{Event/target}} which is an {{Element}} whose 'animation-trigger-name'
+property matches the trigger's associated [=animation trigger name=].
```

Comment:
In the web-animations spec, triggers should not be associated by name. If you're programmatically constructing a trigger you can construct a trigger for a particular element, and add the animation to that trigger, all without using names which could accidentally result in triggering unrelated animations.